target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -42,6 +42,10 @@ ramips_setup_interfaces()
 	asiarf,ap7621-001)
 		ucidef_add_switch "switch0" "0:lan" "4:wan" "6@eth0"
 		;;
+	adslr,g7)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"
```

Comment:
This is duplicated with the following entry:
```sh
	3g-6200n|\
	ai-br100|\
	alfa-network,ac1200rm|\
	mediatek,ap-mt7621a-v60|\
	xzwifi,creativebox-v1|\
	d240|\
	db-wrt01|\
	dir-300-b7|\
	dir-320-b1|\
	dir-610-a1|\
	dir-615-h1|\
	dir-810l|\
	dlink,dwr-116-a1|\
	dlink,dwr-921-c1|\
	dlink,dwr-922-e2|\
	ew1200|\
	firewrt|\
	hc5661a|\
	hc5962|\
	hlk-rm04|\
	k2p|\
	kn|\
	kn_rc|\
	mac1200rv2|\
	miwifi-mini|\
	miwifi-nano|\
	mt7621|\
	mt7628|\
	mtc,wr1201|\
	mzk-750dhp|\
	mzk-w300nh2|\
	nixcore-x1-8M|\
	nixcore-x1-16M|\
	oy-0001|\
	pbr-m1|\
	psg1208|\
	psg1218a|\
	r6220|\
	netgear,r6350|\
	rt-n12p|\
	sap-g3200u3|\
	sk-wb8|\
	telco-electronics,x1|\
	totolink,a7000r|\
	totolink,lr1200|\
	unielec,u7621-06-256m-16m|\
	unielec,u7621-06-512m-64m|\
	vr500|\
	wavlink,wl-wn570ha1|\
	wavlink,wl-wn575a3|\
	wf-2881|\
	whr-g300n|\
	mqmaker,witi-256m|\
	mqmaker,witi-512m|\
	wndr3700v5|\
	youku-yk1|\
	youku,yk-l2|\
	zbt-ape522ii|\
	zbt-we1326|\
	zbtlink,zbt-we826-e|\
	zbtlink,zbt-we3526|\
	zbt-we826-16M|\
	zbt-we826-32M|\
	zbt-wg2626|\
	zbt-wg3526-16M|\
	zbt-wg3526-32M|\
	zbt-wr8305rt)
		ucidef_add_switch "switch0" \
			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"
		;;
```
Just add your device here, instead of adding a duplicated case.

Reply:
After removing this option, the switch "switch0" has an unknown topology and the VLAN settings may be incorrect. Wan mouth is gone

Reply:
噗…
我想说的是，把你的设备加到这个已有的case里面去，不是让你直接删了啊

Reply:
Code not deleted

ramips_setup_interfaces()
{
	local board="$1"

	case $board in
	11acnas|\
	d-team,newifi-d2|\
	dir-615-h1|\
	w2914nsv2|\
	zbt-we2026)
		ucidef_add_switch "switch0" \
			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "4:wan:5" "6@eth0"
		;;
	asiarf,ap7621-001)
		ucidef_add_switch "switch0" "0:lan" "4:wan" "6@eth0"
		;;
	adslr,g7|\
	3g150b|\
	3g300m|\
	a5-v11|\
	all0256n-4M|\
	all0256n-8M|\
	all5002|\
	all5003|\
	bocco|\
	broadway|\``

Reply:
02network代码只删除了
ucidef_add_switch "switch0" \
			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"

这个机器有点特殊，我不指定端口，他就干脆不显示了，我也很无语

Reply:
我等下把代码更新上来，你们看下，我已经按照大佬你说的修改了

Reply:
Could you rebase onto master and push your changes? I'll try to figure out what's wrong here.

Reply:
ok